Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Hidden_User_Created_Test.yml

```diff
@@ -1,1116 +1,3103 @@
-      type: regular
-      version: -1
-    taskid: cad56c70-2a13-4d13-8cd8-e7418afafb3a
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1098,
+          "y": 1052
+        }
+      }
+    note: false
     timertriggers: []
-    type: regular
-    view: "{\n  \"position\": {\n    \"x\": 1100,\n    \"y\": 1100\n  }\n}"
-  '28':
-    continueonerrortype: ''
-    id: '28'
     ignoreworker: false
-    isautoswitchedtoquietmode: false
+    skipunavailable: false
+    quietmode: 0
     isoversize: false
+    isautoswitchedtoquietmode: false
+  "28":
+    id: "28"
+    taskid: c452d4f0-e896-4fde-8b44-be5369a36897
+    type: regular
+    task:
+      id: c452d4f0-e896-4fde-8b44-be5369a36897
+      version: -1
+      name: Extract password expiration flag
+      description: Extracts and saves the PASSWORDEXPIRES value of the locally created user from the results of the Powershell script execution.
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
     nexttasks:
       '#none#':
-      - '9'
-    note: false
-    quietmode: 0
+      - "9"
     scriptarguments:
       key:
         simple: LocalUserPasswordStatus
       value:
         complex:
+          root: Core.ScriptResult.results.[0].command_output
           filters:
-          - - left:
-                iscontext: true
+          - - operator: containsGeneral
+              left:
                 value:
                   simple: Core.ScriptResult.results.[0].command_output
-              operator: containsGeneral
+                iscontext: true
               right:
                 value:
                   simple: Password expires
-          root: Core.ScriptResult.results.[0].command_output
     separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1098,
+          "y": 1286
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
     skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "30":
+    id: "30"
+    taskid: 85e71fec-a160-4cfe-aa82-648feea810b3
+    type: condition
     task:
-      brand: ''
-      description: Extracts and saves the PASSWORDEXPIRES value of the locally created
-        user from the results of the Powershell script execution.
-      id: debbbea1-ba6b-4627-8d3b-a22bcc475682
-      iscommand: false
-      name: Extract password expiration flag
-      scriptName: SetAndHandleEmpty
-      type: regular
+      id: 85e71fec-a160-4cfe-aa82-648feea810b3
       version: -1
-    taskid: debbbea1-ba6b-4627-8d3b-a22bcc475682
-    timertriggers: []
-    type: regular
-    view: "{\n  \"position\": {\n    \"x\": 1100,\n    \"y\": 1260\n  }\n}"
-  '3':
-    continueonerrortype: ''
-    id: '3'
-    ignoreworker: false
-    isautoswitchedtoquietmode: false
-    isoversize: false
+      name: Analyst review - disable suspicious user?
```

Comment:
If from the verdict we get ELSE, ELSE, ELSE, and another ELSE or AD is not available, we will suggest to disable the user, even though we had no findings. Is this intended?

Reply:
I suggest in general to check the flow what happens of XQL doesn't results for some of the queries, or if AD, Slack/Teams are disabled